user-service/README.md

```diff
@@ -100,6 +100,24 @@ curl -X POST user-service/api/user -H "x-request-token: test-request-token" -H "
 }'
 ```
 
+## User pre-population / Cache Warming
+
+This service can be populated with a list of initial users at runtime.
+By specifying all the configuration and pre-population factory values, any user information in the configuration list will be added to the service at runtime.
+This includes being added to the cache, so that the basic `NullUserService` implementation, coupled with the `UserServiceCachingProxy`, acts solely using the cache.
+
+**Note:** This means that after the cache eviction TTL (Time To Live), the User Service will no longer contain any user information.
```

Comment:
add where it can be found:
```
**Note:** This means that after the cache eviction TTL (Time To Live), found in [application.yaml(application.yaml) the User Service will no longer contain any user information.
```

Reply:
Updated